### PR TITLE
FHAC-717: Proxy config cleanup

### PR DESCRIPTION
**Pull request by christophermay07**
Cosmetic updates to make reading / modifying proxy configurations easier.

### CODE REVIEW

**Comment by christophermay07** [comment]
Assigning @alameluchidambaram and @sailajaa for review.

File: Product/Production/Common/Properties/src/main/resources/AdapterAuthenticationProxyConfig.xml [hunk 1]
```diff
@@ -1,35 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beans xmlns="http://www.springframework.org/schema/beans"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.5.xsd"
+       xsi:schemaLocation="
+        http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.5.xsd"
```
Comment (alameluchidambaram) [inline Product/Production/Common/Properties/src/main/resources/AdapterAuthenticationProxyConfig.xml]: The spring-beans-2.5.xsd is changed to 3.0 in other places. Any specific reasons not changed?

Comment (christophermay07) [inline Product/Production/Common/Properties/src/main/resources/AdapterAuthenticationProxyConfig.xml]: Because I missed it :sweat:

**Comment by alameluchidambaram** [comment]
I have reviewed this PR and good with merge.

**Comment by sailajaa** [comment]
Reviewed and good with PR merge.

**Pull request merged**